domain/src/main/java/com/example/domain/useCase/weatherData/DeleteOldWeatherDataFromEntityUseCase.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.domain.useCase.weatherData
+
+import com.example.domain.repositories.DeleteOldWeatherDataRepository
+
+class DeleteOldWeatherDataFromEntityUseCase(
+    private val deleteOldWeatherDataRepository: DeleteOldWeatherDataRepository
+) {
+    operator fun invoke(cityId: Int) {
```

Comment:
We always pass cityDataCache.loadCityId() as a cityId.  Let's get rid of input parameter cityId and move the getting id from cityDataCache inside this class.

Reply:
Fixed